### ISSUE TITLE
Codegen for extinst.glsl.std.450.grammar.json

### ISSUE THREAD

**Issue by MaikKlein** [google/rspirv#24]
https://github.com/KhronosGroup/SPIRV-Headers/blob/master/include/spirv/1.2/extinst.glsl.std.450.grammar.json

Should this also be included in the spirv-headers?

**Comment by MaikKlein** [comment]
Oops sorry I didn't see it https://github.com/google/rspirv/blob/master/rspirv/grammar/glsl_std_450.rs

**Comment by MaikKlein** [comment]
I have to reopen this issue. How do you lookup an opcode? I only see https://github.com/google/rspirv/blob/master/rspirv/grammar/glsl_std_450.rs

But that doesn't help me if I want to know the opcode for `Sqrt`. In my project, I am currently defining them manually like this

```
#[repr(u32)]
pub enum GlslExtId {
    Round = 1,
    Sqrt = 31,
}
```
Because that is the id that is being used in [ext_inst](https://docs.rs/rspirv/0.4.0/rspirv/mr/struct.Builder.html#method.ext_inst)

**Comment by antiagainst** [comment]
Acknowledged. There are no way to get the opcode number from the symbolic opcode right now. Will add methods soon. :)

**Comment by antiagainst** [comment]
Should be addressed by https://github.com/google/rspirv/commit/1e180a99034b900f6a35bc7b48c02ea6fa481f2a.

I'll close this. Let me know if more actions are needed. Thanks!

**Comment by antiagainst** [comment]
I'll push a new release soon.

**Comment by MaikKlein** [comment]
@antiagainst Thanks! Could also push a new version of `rspirv`?

**Comment by antiagainst** [comment]
@MaikKlein: published v0.5.0. Thanks! :)
